scripts/restore

```diff
@@ -52,31 +52,28 @@ ynh_script_progression --message="Restoring the NGINX configuration..." --weight
 
 ynh_restore_file --origin_path="/etc/nginx/conf.d/$domain.d/$app.conf"
 
-#=================================================
-# RESTORE THE APP MAIN DIR
-#=================================================
-ynh_script_progression --message="Restoring Moodle main directory..." --weight=1
-
-ynh_restore_file --origin_path=$final_path
-
-ynh_restore_file --origin_path=$data_path
-
 #=================================================
 # RECREATE THE DEDICATED USER
 #=================================================
 ynh_script_progression --message="Recreating the dedicated system user..." --weight=1
 
 # Create the dedicated user (if not existing)
-ynh_system_user_create --username=$app
+ynh_system_user_create --username=$app --home_dir="$final_path"
 
 #=================================================
-# RESTORE USER RIGHTS
+# RESTORE THE APP MAIN DIR
 #=================================================
-ynh_script_progression --message="Restoring user rights..."
+ynh_script_progression --message="Restoring Moodle main directory..." --weight=1
+
+ynh_restore_file --origin_path=$final_path
+
+chmod 750 "$final_path"
+chmod -R o-rwx "$final_path"
+chown -R $app:www-data "$final_path"
+
+ynh_restore_file --origin_path=$data_path
```

Comment:
Would be cool to use `--is_big` during backup and `--not_mandatory` during restore.
Doing that `$data_path` won't be backup during ynh_backup_before_upgrade in the upgrade script; making upgrade quicker